UI/src/app/components/toolbar/toolbar.component.ts

```diff
@@ -53,6 +53,10 @@ export class ToolbarComponent implements OnInit {
   }
 
   resetSourceAndTarget() {
-    this.commonUtilsService.openResetWarningDialog(true);
+    this.commonUtilsService.openResetWarningDialog('All mappings will be lost. Do you want to save created mappings?',
```

Comment:
Let's pass object instead of many comma-separated values

Reply:
done